vendor-extra/JatsContentBundle/src/Handler/JatsContentHandler.php

```diff
@@ -46,11 +48,22 @@ public function handle(Element $documentElement, array $context) : array
 
         $contentHeader = $this->converter->convert($front, '@LiberoPatterns/content-header.html.twig', $context);
 
+        $content = [$contentHeader];
```

Comment:
Not blocking: since `$contentHeader` is not being used again might be cleaner to?:

```
$content = [
    $this->converter->convert($front, '@LiberoPatterns/content-header.html.twig', $context),
];
```

Reply:
It's used in https://github.com/libero/browser/blob/b6ebf895424c7b62a0587188f5b26343bb3dc998/vendor-extra/JatsContentBundle/src/Handler/JatsContentHandler.php#L65

Reply:
Ignore me!